src/qmapshack/poi/IPoi.h

```diff
@@ -48,6 +48,11 @@ class IPoi : public IDrawObject
 
     bool activated() const {return isActivated;}
 
+    virtual void addTreeWidgetItems(QTreeWidget* widget) = 0;
+
+public slots:
+    virtual void checkedStateChanged(QTreeWidgetItem*item, int columnChanged) = 0;
```

Comment:
Stick to the name pattern: slots names start with `slot...` signals with `sig...` It really helps.

Reply:
Yep, forgot about that :D